packages/ui/src/proposals/hooks/useVotingRounds.ts

```diff
@@ -28,7 +28,7 @@ export const useVotingRounds = (votes: ProposalVote[] = [], updates: ProposalSta
   const councilSize = useCouncilSize()
 
   const voteRounds: (Omit<VotingRound, 'count'> & { total?: number })[] = useMemo(() => {
-    const decidingCount = updates.filter(({ status }) => status === 'deciding').length || 1
+    const decidingCount = updates.findIndex(({ status }) => status === 'deciding') + 1 || 1
```

Comment:
I think this won't work.

To see how proposalStatusUpdates works check e.g:
```gql
{
  proposalByUniqueInput (where: { id: "63" }) {
    proposalStatusUpdates {
      newStatus { __typename }
    }
  }
}
```

You could use the index based on the assumption that only dormant and deciding stage alternate up to the last deciding stage. But that would be very unclear.

So I thinks this should be:
```suggestion
    const decidingCount = updates.filter(({ status }) => status === 'deciding').length + 1
```
Same as in `@/proposals/components/ProposalStages`

Reply:
If the suggested code is implemented, `decidingCount` will be 1 forever because there is only one update with the status `deciding` at any given time. 
As soon as a new deciding stage is entered, the former `deciding` stage goes into `dormant`.

Reply:
No `update` keeps track of all stage changes.

Please try the query I wrote above on https://query.joystream.org/graphql

Because this proposal was voted on 3 times, it shows 2 `ProposalStatusDeciding` "newStatus" (as it doesn't include the initial deciding stage).